apps/super-app/src/common/components/navbar/NavBar.css

```diff
@@ -0,0 +1,44 @@
+@import url("https://fonts.googleapis.com/css?family=Hind");
+a:visited {
+  text-decoration: none;
+}
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0px 32px 10px;
+  gap: 56px;
+  position: absolute;
+  width: 360px;
+  height: 72px;
+  left: 16px;
+  top: 104px;
+  background-color: #f3fafc;
+}
+
+.navbar-icon {
+  display: inline-block;
```

Comment:
Let's use `display: flex` instead; it's easier to control the formatting than by manipulating line heights in inline-block elements :)